pi_face_tracker/nodes/face_tracker.py

```diff
@@ -1,20 +1,18 @@
 #!/usr/bin/env python
```

Comment:
I think you want to change the above to `python3`  since usually, plain `python` invokes python2.

Reply:
So for the python3: that is handled by linking python to python3 in /usr/lib/???, since that first line is ubiquitous. 
At least for now, I didn't want to go and change ALL the files!
As for whether to create a "new" file and repoint: well, do we want the old code hanging about? Perhaps it would be good to tag the repo at this point, so we can checkout this "last" revision of pre-noetic. But it seems to me, we should be modifying the existing code "in place" or else we'll just have a lot of preserved junk. It isn't going to ever run again, since opencv is now CV2, and we're on python3.
So I'd say YES I'm sure, but if you can tag the repo, that would be good?

Reply:
(perhaps one assumption I'm making: the code base should all run with python3 correct?): I mean, we don't run a mixture I hope.

Reply:
that is to say, the existing python 1 & 2 code is broken  That is how I interpreted it.